site/content/docs/03-template-syntax.md

```diff
@@ -663,7 +663,7 @@ A `<select>` value binding corresponds to the `value` property on the selected `
 
 ---
 
-A `<select multiple>` element behaves similarly to a checkbox group.
+A `<select multiple>` element behaves similarly to a checkbox group. The bound variable is an array with an entry containing the `value` property of each selected `<option>`.
```

Comment:
As it's written now it sounds like there's a single entry, but I'm assuming there are multiple entries in the array
```suggestion
A `<select multiple>` element behaves similarly to a checkbox group. The bound variable is an array with an entry corresponding to the `value` property of each selected `<option>`.
```